schema/README.md

```diff
@@ -0,0 +1,32 @@
+# JSON schema
+
+## Overview
+
+This directory contains the [JSON Schema](http://json-schema.org/) for
+validating the `config.json` of this container runtime specification.
+
+The layout of the files is as follows:
+* [schema.json](./schema.json) - the primary entrypoint for the whole schema document
```

Comment:
muscle memory. fixed.

On Wed, Apr 13, 2016 at 1:43 PM, W. Trevor King <notifications@github.com>
wrote:

> In schema/README.md
> <https://github.com/opencontainers/runtime-spec/pull/360#discussion_r59592283>
> :
>
> > @@ -0,0 +1,32 @@
> > +# JSON schema
> > +
> > +## Overview
> > +
> > +This directory contains the [JSON Schema](http://json-schema.org/) for
> > +validating the `config.json` of this container runtime specification.
> > +
> > +The layout of the files is as follows:
> > +* [schema.json](./schema.json) - the primary entrypoint for the whole schema document
>
> Pedantic nit: the other link targets don't have a leading ./, so maybe
> remove this one for consistency?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/360/files/57d2484193eb1631c186f308cf65c1f0fac43952#r59592283>
>